dipy/align/streamlinear.py

```diff
@@ -828,18 +837,17 @@ def progressive_slr(static, moving, metric, x0, bounds,
 
     return slm
 
-
-def slr_with_qb(static, moving,
-                x0='affine',
-                rm_small_clusters=50,
-                maxiter=100,
-                select_random=None,
-                verbose=False,
-                greater_than=50,
-                less_than=250,
-                qb_thr=15,
-                nb_pts=20,
-                progressive=True, num_threads=None):
+def slr_with_qbx(static, moving,
+                 x0='affine',
+                 rm_small_clusters=50,
+                 maxiter=100,
+                 select_random=None,
+                 verbose=False,
+                 greater_than=50,
+                 less_than=250,
+                 qbx_thr=[40, 30, 20, 15],
+                 nb_pts=20,
+                 progressive=True, rng=None, num_threads=None):
     """ Utility function for registering large tractograms.
```

Comment:
Can you add `maxiter`, `progressive`, `nb_pts`, `qbx_thr`, `less_than`, `greater_than` in the docstring ?